WalletWasabi.Backend/Global.cs

```diff
@@ -37,7 +37,8 @@ public Global(string dataDir)
 
 	public HostedServices HostedServices { get; }
 
-	public IndexBuilderService? IndexBuilderService { get; private set; }
+	public IndexBuilderService? SegwitTaprootIndexBuilderService { get; private set; }
+	public IndexBuilderService? TaprootIndexBuilderService { get; private set; }
```

Comment:
Why are these nullables if they can't/shouldn't be null?

Reply:
They can be null. Concepts here were implemented before nullability in C# existed. Also I would strongly prefer not refactoring anything problematic like that in this PR.

Reply:
Fixed.

Reply:
There was a conflict after the @kiminuo refactoring in the `Global` class so, fixing the conflict was not optional.